test/test_utils.dart

```diff
@@ -207,6 +207,22 @@ class AsyncCommand extends Command {
   Future run() => Future.value().then((_) => hasRun = true);
 }
 
+class AllowAnythingCommand extends Command {
+  var hasRun = false;
+
+  @override
+  final name = 'allowAnything';
+
+  @override
+  final description = 'A command using allowAnything.';
+
+  @override
+  final argParser = ArgParser.allowAnything();
+
+  @override
+  Future run() => Future.value().then((_) => hasRun = true);
```

Comment:
just use `async` here?

Reply:
you would need to `await null` before this line if you do that.

Reply:
Why are we relying on there being a microtask before setting the value?

Reply:
This is taken from the command impl. above, but it looks like it may not be necessary.

Reply:
(changed to just `void run() { hasRun = true; }`)